test/testenv/verificationutils.go

```diff
@@ -375,3 +384,43 @@ func VerifyIndexerClusterScalingUp(deployment *Deployment, testenvInstance *Test
 		return idxc.Status.Phase
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(splcommon.PhaseScalingUp))
 }
+
+// VerifyStandaloneUpdating ensures Standalone is scaling up
+func VerifyStandaloneUpdating(deployment *Deployment, deploymentName string, standalone *enterprisev1.Standalone, testenvInstance *TestEnv) {
+	gomega.Eventually(func() splcommon.Phase {
+		err := deployment.GetInstance(deploymentName, standalone)
+		if err != nil {
+			return splcommon.PhaseError
+		}
+		testenvInstance.Log.Info("Waiting for standalone STATUS to be updating", "instance", standalone.ObjectMeta.Name, "Phase", standalone.Status.Phase)
+		DumpGetPods(testenvInstance.GetName())
+		return standalone.Status.Phase
+	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(splcommon.PhaseUpdating))
+}
+
+// VerifyCPULimits verifies value of CPU limits is as expected
+func VerifyCPULimits(deployment *Deployment, ns string, podName string, expectedCPULimits string) {
```

Comment:
nit: Maybe we can move the common code to get pod details into a single method. This is being used in multiple places. Can be taken up later in a refactor effort.